controllers/auth.go

```diff
@@ -459,10 +459,23 @@ func (c *ApiController) Login() {
 			}
 
 			if isSigninViaLdap {
-				user, err = object.GetUserByFields(authForm.Organization, authForm.Username)
+				ldap, err := object.GetLdap(authForm.LdapId)
+				if err != nil {
+					record.AddReason(fmt.Sprintf("Get Ldap error: %s", err.Error()))
+				}
+				if ldap == nil {
+					record.AddReason(fmt.Sprintf("Cannot get Ldap \"%s\"", authForm.LdapId))
+				}
+
+				if ldap != nil && ldap.EnableCaseInsensitivity {
```

Comment:
What you think about simplify if expression
```
if ldap != nil {
 if ldap.EnableCaseInsensitivity {
  } else {
  } 
}
```
?

Reply:
I wanted to avoid unnecessary nesting, but maybe it would be better here

Reply:
done